test_build.py

```diff
@@ -86,6 +86,10 @@
 import fastText
 print("fastText ok")
 
+import pycuda
```

Comment:
It would be good to check that both TF and PyTorch can connect to a GPU if it's available. It seems that for PyTorch the [recommended way](https://discuss.pytorch.org/t/pytorch-installation-with-gpu-support/9626/3) is to call `torch.cuda.is_available()`, and for TF [I guess](https://www.tensorflow.org/programmers_guide/using_gpu) it's to run a session with `log_device_placement=True` and grep the output for `GPU`…?

Reply:
We could also use a `with tf.device('/cpu:0'):` block to check that a TensorFlow hello-world can execute on a CPU-only setup. I'm not sure how to manually choose a device with PyTorch, I could only find an [open issue](https://github.com/pytorch/pytorch/issues/1668) about it.